src/backend/expungeservice/endpoints/auth.py

```diff
@@ -71,9 +71,8 @@ def authorized(f, admin_required, *args, **kwargs):
         if admin_required and not user_data['admin']:
             error(403, 'Logged in user not admin')
 
-        # no endpoint code uses the logged in user_id so this is commented for
-        # now. This may change.
-        # g.logged_in_user_id = user_data['user_id']
+        g.logged_in_user_id = user_data['user_id']
+        g.logged_in_user_is_admin = user_data['admin']
```

Comment:
hmmm lets talk Wednesday about the potential security issue here. It's possibly a low security issue, however there are "best" practices to use to check if current user is an admin.

Reply:
I did some more reading up on JWT's and this is actually the common way to use them although it is a bit strange to me. Anyway's not super important, but we can talk about it if there is time.